content/post/12-zappos-offers-new-hires/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Zappos Offers New Hires $3,000 to Quit After 4 Weeks"
+date: 2010-11-16T00:00:00-07:00
+draft: false
+categories:
+- Entrepreneurship
+- Zen
```

Comment:
All the categories are lower case, so replace with:

```yaml
categories:
- Entrepreneurship
- Zen
```

Reply:
Got it.